rfcs/text/0041-asset-integration.md

```diff
@@ -165,14 +175,540 @@ There are many sources of asset inventory repositories. In the mid-term, we are
+      tag: rename_user_profile_state
+      ignore_missing: true
+  - rename:
+      field: okta.profile.zipCode
+      target_field: entityanalytics_okta.user.profile.zip_code
+      tag: rename_user_profile_zip_code
+      ignore_missing: true
+  - rename:
+      field: okta.profile.countryCode
+      target_field: entityanalytics_okta.user.profile.country_code
+      tag: rename_user_profile_country_code
+      ignore_missing: true
+  - rename:
+      field: okta.profile.postalAddress
+      target_field: entityanalytics_okta.user.profile.postal_address
+      tag: rename_user_profile_postal_address
+      ignore_missing: true
+  - rename:
+      field: okta.profile.preferredLanguage
+      target_field: entityanalytics_okta.user.profile.preferred_language
+      tag: rename_user_profile_preferred_language
+      ignore_missing: true
+  - rename:
+      field: okta.profile.locale
+      target_field: entityanalytics_okta.user.profile.locale
+      tag: rename_user_profile_locale
+      ignore_missing: true
+  - rename:
+      field: okta.profile.timezone
+      target_field: entityanalytics_okta.user.profile.timezone
+      tag: rename_user_profile_timezone
+      ignore_missing: true
+  - rename:
+      field: okta.profile.userType
+      target_field: entityanalytics_okta.user.profile.user_type
+      tag: rename_user_profile_user_type
+      ignore_missing: true
+  - set:
+      field: user.profile.type
+      copy_from: entityanalytics_okta.user.profile.user_type
+      tag: set_user_profile_type
+      ignore_empty_value: true
+  - rename:
+      field: okta.profile.employeeNumber
+      target_field: entityanalytics_okta.user.profile.employee_number
+      tag: rename_user_profile_employee_number
+      ignore_missing: true
+  - append:
+      field: related.user
+      value: '{{{entityanalytics_okta.user.profile.employee_number}}}'
+      tag: append_user_profile_employee_number_into_related_user
+      allow_duplicates: false
+      if: ctx.entityanalytics_okta?.user?.profile?.employee_number != null
+  - set:
+      field: user.profile.id
+      copy_from: entityanalytics_okta.user.profile.employee_number
+      tag: set_user_profile_id
+      ignore_empty_value: true
+  - rename:
+      field: okta.profile.costCenter
+      target_field: entityanalytics_okta.user.profile.cost_center
+      tag: rename_user_profile_cost_center
+      ignore_missing: true
+  - set:
+      field: asset.costCenter
+      copy_from: entityanalytics_okta.user.profile.cost_center
+      tag: set_asset_costCenter
+      ignore_empty_value: true
+  - rename:
+      field: okta.profile.organization
+      target_field: entityanalytics_okta.user.profile.organization
+      tag: rename_user_profile_organization
+      ignore_missing: true
+  - set:
+      field: user.organization.name
+      copy_from: entityanalytics_okta.user.profile.organization
+      tag: set_user_profile_organization
+      ignore_empty_value: true
+
+```
+
+ 
+#### AzureAD Hosts
```

Comment:
I would prefer to drop this. In fact, I'd prefer we simply link to the source for these "real world examples", because they could change at any time. In fact...the Okta one has changed since this RFC was written!

In the case of Okta, the link to that source code would be here: https://github.com/elastic/integrations/blob/main/packages/entityanalytics_okta/data_stream/user/elasticsearch/ingest_pipeline/default.yml